### PR TITLE
Courier Tracking

### PR DESCRIPTION
**Pull request by shadowcodex**
This pull request is to track commits related to the courier tracking module. Currently it is not ready to merge, I will comment when I believe it is ready to merge.

### CODE REVIEW

**Comment by MiliasColds** [comment]
Looks good but I don't think merging is good until is functioning since this repo will be a reflection of prod code

**Comment by shadowcodex** [comment]
Correct. This pull request is for tracking and discussion.

File: courier/views.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+logger = logging.getLogger(__name__)
+
+
+# Create your views here.
+
+@login_required
+@members_and_blues()
+def index(request):
+    logger.debug("Courier index called by user %s" % request.user)
+    context = {"open": 0, "inProgress": 0, "completed": 0}
```
Comment (shadowcodex) [inline courier/views.py:25]: need to figure out how to actually pull this data.

**Comment by shadowcodex** [comment]
Closing this PR. Rebasing my fork, but you can find all these changes here: https://github.com/shadowcodex/courier

**Pull request closed without merging**